apps/admin/frontend/src/screens/election_screen.tsx

```diff
@@ -66,8 +66,8 @@ export function ElectionScreen(): JSX.Element {
         {isElectionManagerAuth(auth) && (
           <React.Fragment>
             <P>
-              Save the Ballot Package to USB to configure VxCentralScan or
-              VxScan.
+              Save the election package to the USB drive to configure
+              VxCentralScan or VxScan.
             </P>
```

Comment:
I wonder if we should add VxMark to this list, because it now requires an election package, or if we should be non-specific, as different customers will have different machines.

Reply:
@mattroe thoughts?

Reply:
"VxSuite components"?

Reply:
Updated